### ISSUE TITLE
Possible issue with Microsoft.Extensions.Logging v5 in release 1.1.0

### ISSUE THREAD

**Issue by stage88** [awslabs/amazon-qldb-driver-dotnet#57]
Hi guys,

There is a possible issue with using the Microsoft.Extensions.Logging package v5  in https://github.com/awslabs/amazon-qldb-driver-dotnet/blob/master/Amazon.QLDB.Driver/Amazon.QLDB.Driver.xml. In my case, after upgrading to Amazon.QLDB.Driver v1.1.0 I get compiler errors:

```
1>D:\SomeProject\Project.csproj : error NU1605: Detected package downgrade: Microsoft.Extensions.DependencyInjection from 5.0.0 to 3.1.11. Reference the package directly from the project to select a different version. 
1>D:\SomeProject\Project.csproj : error NU1605: Project -> Amazon.QLDB.Driver 1.1.0 -> Microsoft.Extensions.Logging 5.0.0 -> Microsoft.Extensions.DependencyInjection (>= 5.0.0) 
1>D:\SomeProject\Project.csproj : error NU1605: Project -> Microsoft.Extensions.DependencyInjection (>= 3.1.11)
```

My code runs on AWS Lambda .NET 3.1 run time and I don't yet have the option of using .NET 5. I am not sure how best to solve this problem, but for now I will stay on Amazon.QLDB.Driver 1.0.1

**Comment by yoroto** [comment]
Hi @stage88 

I will get back to you soon.

**Comment by byronlin13** [comment]
Hi @stage88 , could you give me steps to reproduce this issue?

**Comment by stage88** [comment]
Hi @byronlin13,

Sorry, I should have included these when I created the issue. Steps to reproduce are below.

1. In Visual Studio, create a new class library project for .NET Core. The project file should be like this targeting .NET Core 3.1
```
<Project Sdk="Microsoft.NET.Sdk">
  <PropertyGroup>
    <TargetFramework>netcoreapp3.1</TargetFramework>
  </PropertyGroup>
</Project>
```
2. Update the project file and add these package references, the file should look like this:
```
<Project Sdk="Microsoft.NET.Sdk">
  <PropertyGroup>
    <TargetFramework>netcoreapp3.1</TargetFramework>
  </PropertyGroup>
  <ItemGroup>
    <PackageReference Include="Amazon.QLDB.Driver" Version="1.0.1" />
    <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="3.1.11" />
  </ItemGroup>
</Project>
```
3. Update the project file to use the latest version of the QLDB driver:
```
<Project Sdk="Microsoft.NET.Sdk">
  <PropertyGroup>
    <TargetFramework>netcoreapp3.1</TargetFramework>
  </PropertyGroup>
  <ItemGroup>
    <PackageReference Include="Amazon.QLDB.Driver" Version="1.1.0" />
    <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="3.1.11" />
  </ItemGroup>
</Project>
```
4. Try and build, you should get a build error:
```
NuGet package restore failed. Please see Error List window for detailed warnings and errors.
1>------ Build started: Project: ClassLibrary1, Configuration: Debug Any CPU ------
1>D:\Temp\ClassLibrary1\ClassLibrary1\ClassLibrary1.csproj : error NU1605: Detected package downgrade: Microsoft.Extensions.DependencyInjection from 5.0.0 to 3.1.11. Reference the package directly from the project to select a different version. 
1>D:\Temp\ClassLibrary1\ClassLibrary1\ClassLibrary1.csproj : error NU1605:  ClassLibrary1 -> Amazon.QLDB.Driver 1.1.0 -> Microsoft.Extensions.Logging 5.0.0 -> Microsoft.Extensions.DependencyInjection (>= 5.0.0) 
1>D:\Temp\ClassLibrary1\ClassLibrary1\ClassLibrary1.csproj : error NU1605:  ClassLibrary1 -> Microsoft.Extensions.DependencyInjection (>= 3.1.11)
1>Done building project "ClassLibrary1.csproj" -- FAILED.
========== Build: 0 succeeded, 1 failed, 0 up-to-date, 0 skipped ==========
```

Hopefully that helps. Also some context: at this stage I am unable to update from .NET 3.1.* set of Microsoft packages, my team is currently pining to 3.1.11 and .NET Core 3.1

**Comment by aasadsan** [comment]
Hi @stage88, this is fixed in v1.1.1 of the driver.

Closing the ticket now, but let us know if you still have the same problem with the new fix.

**Comment by stage88** [comment]
Thanks for the quick update, our builds are now compiling with the latest version.
